.github/workflows/unit-tests.yml

```diff
@@ -0,0 +1,28 @@
+#
+#  Copyright (c) 2012-2020 Red Hat, Inc.
+#    This program and the accompanying materials are made
+#    available under the terms of the Eclipse Public License 2.0
+#    which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+#  SPDX-License-Identifier: EPL-2.0
+#
+#  Contributors:
+#    Red Hat, Inc. - initial API and implementation
+name: Chectl
+on: [pull_request, push]
```

Comment:
if we enforce that we must submit a PR for every change, do we need to run tests on every push too?

Reply:
oh, my mistake... Only in PRs